readme.rst

```diff
@@ -229,6 +229,32 @@ your doc files were placed (repository root or the ``docs`` directory):
    make install
    make html
 
+Build a PDF
+^^^^^^^^^^^
+
+The documentation can build a PDF locally with the following command:
```

Comment:
I don't think the documentation can do anything. ;)
Rather "You can build a PDF for your documentation" or "You can build the documentation as a PDF" or something.

Reply:
```suggestion
Build a PDF locally with the following command:
```

Reply:
Yes, you're quite correct, haha.